Sources/XcodeGen/Logger.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  Logger.swift
+//  XcodeGen
+//
+//  Created by Sam Soffes on 11/21/17.
+//
```

Comment:
Could you remove header. I've removed them all in master, and they are all removed automatically now in `make format_code`